src/app.py

```diff
@@ -12,6 +12,7 @@ def format(self, record):
             record.corr = request.headers.get('fiware_correlator', None)
             record.remote_addr = request.remote_addr
             record.srv = request.headers.get('fiware-service', None)
+            record.tenant = request.headers.get('ngsild-tenant', None)
```

Comment:
as said the simplest way is to reuse srv and implement a logic as that assign (if available)
ngsild-tenant, if not fiware-service.

Reply:
I understood the implementation that you were looking for by this comment. I have done the required changes to reuse srv and tested. ngsild-tenant is given priority over fiware-service.